hc/settings.py

```diff
@@ -133,7 +135,8 @@
 )
 COMPRESS_OFFLINE = True
 
-EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+EMAIL_BACKEND = "sgbackend.SendGridBackend"
+SENDGRID_API_KEY = os.environ.get('SENDGRID_API_KEY')
```

Comment:
I guess we are using a .env file that we are ignoring, while it should remain ignored, could you post the config on our slack channel so I can have similar config